### ISSUE TITLE
Route Packets with Empty Marti

### ISSUE THREAD

**Issue by jazap** [tkuester/taky#68]
Allow iTAK point dropper sharing

**Comment by skadakar** [comment]
Example COT:

Broadcast to everyone from itak:
```
<event version="2.0" uid="EB77220E-6299-4CA3-95FC-0200BD9FE78A" type="a-u-G" how="h-g-i-g-o" time="2023-01-12T09:53:31.000Z" start="2023-01-12T09:53:31.000Z" stale="2023-01-12T09:55:31.000Z">
	<point lat="54.338986" lon="9.755263" hae="0.0" ce="0.0" le="0.0"/>
	<detail>
		<contact callsign="poop"/>
		<precisionlocation geopointsrc="???" altsrc="???"/>
		<status readiness="true"/>
		<archive/>
		<link uid="1874F828-5CF1-4289-B6C0-D4F6ABFB0B4D" production_time="2023-01-12T09:53:31Z" type="a-f-G-U-C" parent_callsign="FM05-iOS" relation="p-p"/>
		<usericon iconsetpath="COT_MAPPING_2525B/a-u/a-u-G"/>
		<color argb="-1"/>
		<marti/>
		<remarks/>
	</detail>
</event>
```

Send to two specific people from itak:
```
<event version="2.0" uid="0B90E184-C49B-4321-8912-4B3FC628328E" type="a-u-G" how="h-g-i-g-o" time="2023-01-12T10:06:26.000Z" start="2023-01-12T10:06:26.000Z" stale="2023-01-12T10:08:26.000Z">
	<point lat="53.601004" lon="9.556352" hae="0.0" ce="0.0" le="0.0"/>
	<detail>
		<contact callsign="testmarker"/>
		<precisionlocation geopointsrc="???" altsrc="???"/>
		<status readiness="true"/>
		<archive/>
		<link uid="1874F828-5CF1-4289-B6C0-D4F6ABFB0B4D" production_time="2023-01-12T10:06:26Z" type="a-f-G-U-C" parent_callsign="FM05-iOS" relation="p-p"/>
		<usericon iconsetpath="COT_MAPPING_2525B/a-u/a-u-G"/>
		<color argb="-1"/>
		<marti>
			<dest uid="ASN-TAK-BOT-FAKE-UID"/>
			<dest uid="S-1-5-21-3708732509-2984588875-2455516043-1001"/>
		</marti>
		<remarks/>
	</detail>
</event>
```

**Comment by tkuester** [comment]
Fixed in dfcb731
